### DISCUSSION TITLE
Extracting images from HTML files

### DISCUSSION THREAD

**Discussion by piotrkandziora** [Unstructured-IO/unstructured#2570] [Q&A]
I am considering utilizing GPT-4V to replace the images that are extracted from HTML files. The information can be obtained directly from the code, but I am wondering if it is also possible to obtain it from the partition_html parsing result.

Is it to be implemented or not implemented for some reason?

**Comment by lukajincharadze** [comment]
it remains unknown whether GPT-4V, if released, would possess the capability to parse HTML and replace images extracted from HTML files. Such a feature's feasibility and implementation would hinge on GPT-4V's capabilities and intended functionalities. The decision to implement such functionality would depend on various factors, including its alignment with the intended use cases, user expectations, and potential privacy or ethical considerations. Further details regarding GPT-4V's capabilities and its applicability to HTML parsing and image replacement would need to be provided by the developers or obtained upon its release.
